components/Pagination.js

```diff
@@ -45,6 +46,7 @@ class Pagination extends Component {
     page: this.props.page,
     pageSize: ((this.props.pageSize && this.props.pageSizes.includes(this.props.pageSize)) ?
       this.props.pageSize : this.props.pageSizes[0]),
+    pagesUnknown: this.props.pagesUnknown || false
```

Comment:
For defaultProps, you should be able to write: 
```
pagesUnknown: false
```

Then the user doesn't have to explicitly give the prop a boolean value. Instead, they can write:

```
// pagesUnknown is true
<Pagination pagesUnknown />

// pagesUnknown is false
<Pagination />
```